HW_1/task_8.py

```diff
@@ -0,0 +1,17 @@
+#Нарисовать в консоли ёлку спросив
+#у пользователя количество рядов.
+
+SPASE = ' '
+STAR = '*'
+rows = int(input('Сколько рядов у елки? '))
+spases = rows -1
+stars = 1
+
+for i in range(rows):
+    print(SPASE*spases+STAR*stars+SPASE*spases)
```

Comment:
Про PEP8 не забывайте
```suggestion
    print(SPASE * spases + STAR * stars + SPASE * spases)
```